packages/office-ui-fabric-react/src/components/Theme/ThemePage.tsx

```diff
@@ -115,7 +115,7 @@ export class ThemePage extends React.Component<any, any> {
   private _onColorChanged(index: number, newColor: string) {
     let { colors } = this.state;
     let color = colors[index];
-    let theme: ITheme = {};
+    let theme: Partial<IPalette> = {};
```

Comment:
I believe there's a bug here. Either this is a palette (in which case the theme variable should be renames) or we are in correctly creating an object to pass to loadTheme.